src/ui/zcl_abapgit_gui_view_repo.clas.abap

```diff
@@ -252,9 +254,18 @@ CLASS zcl_abapgit_gui_view_repo IMPLEMENTATION.
       ro_toolbar->add( iv_txt = 'Tag'
                        io_sub = lo_tb_tag ) ##NO_TEXT.
     ELSE.
-      ro_toolbar->add( iv_txt = 'Import ZIP'
-                       iv_act = |{ zif_abapgit_definitions=>c_action-zip_import }?{ lv_key }|
-                       iv_opt = zif_abapgit_definitions=>c_html_opt-strong ).
+      IF mo_repo->has_remote( ) = abap_true.
+        ro_toolbar->add( iv_txt = 'Apply'
+                         iv_act = |{ c_actions-zip_import2_apply }|
+                         iv_opt = zif_abapgit_definitions=>c_html_opt-strong ).
+        ro_toolbar->add( iv_txt = 'Show diff'
```

Comment:
same button as from online can be reused instead of adding another?

Reply:
Do you mean "Show diff"? Yes, but there are a lot of ifs and one try ... I don't think there is a good way to do it without duplication. Unless the position of Show diff is changed both both on and offline. But I think the position is ok now ?

Reply:
I think this can be refactored, but we can do it in a follow up PR